AIX_types.md

```diff
@@ -0,0 +1,66 @@
+AIX Specific Type Documentation
```

Comment:
Sorry that I didn't mention this sooner, but we prefer documentation to be in a single file. Could you move this to a subsection under the main README's reference section? I think it's fine to still have them separate from the rest of the properties though.